.travis.yml

```
@@ -18,7 +18,6 @@ script:
 - git rebase origin/master
 - git log --oneline | head -n 10
 - mdbook build
-after_script:
 - ([ -e /tmp/cache.json ] && mv /tmp/cache.json book/linkcheck/cache.json) || echo "No cache to restore."
```

This spawns a new subshell.
```suggestion
- { [ -e /tmp/cache.json ] && mv /tmp/cache.json book/linkcheck/cache.json; } || echo "No cache to restore."
```